zlp-scheduler/spec/factories.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+    factory :cohort do
+        id {rand(10000)}
+    end
+
+    factory :user do
+        firstname {Faker::Name.name}
+        lastname {Faker::Name.name}
+        email {Faker::Internet.email}
+        password {Faker::Internet.password}
+    end
```

Comment:
Does this factory create valid users? For a user to be valid they have to have @tamu.edu at then of the email.

Reply:
That's a good point. I will update it, and add another scenario about user login with non-tamu email.